docker-compose.local.yml

```diff
@@ -0,0 +1,53 @@
+version: "3"
+
+services:
+  app:
+    image: ssthapa/uipa_backend:0.9
```

Comment:
I don't _love_ having a separate image here - but it's fine. We should have instructions on how to switch this out/publish our own containers though if so

Reply:
I should actually have a 0.8, it's 0.9 because I was using that for testing.  I'll fix that.  The image is generated from the Dockerfile in the top level directory.  We can just change the tag to `uipa_backend:x` to use a locally built copy.   We need to document this though.